unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Rendering/RenderingController.cs

```diff
@@ -89,11 +89,21 @@ private void ActivateRendering_Internal()
         AssetPromiseKeeper_AB.i.useTimeBudget = true;
         AssetPromiseKeeper_AB_GameObject.i.useTimeBudget = true;
 
-        OnRenderingStateChanged?.Invoke(renderingEnabled);
+        CommonScriptableObjects.rendererState.Set(true);
 
         MemoryManager.i.CleanupPoolsIfNeeded(true);
         ParcelScene.parcelScenesCleaner.ForceCleanup();
```

Comment:
What about those dependencies? We can't make them use the `rendererState` variable?